File: Julia_Q4_Sum.java
import java.util.Scanner;
import java.util.Vector;
class HelloWorld {
    public static void main(String[] args) {
        int ans=0, i=0, size =0;
        Scanner input = new Scanner(System.in); 
        Vector<Integer> vec = new Vector<Integer>(size);
        do{
            vec.setSize(size);
            System.out.print("Input " + (i+1) +": ");
            vec.add(input.nextInt());
            i++;
            size++;
            System.out.print("To add more = 1, To print = 0: ");
            ans = input.nextInt();
        }while(ans==1);
        System.out.println("Array Output:");
        for (Integer num : vec) {         
            System.out.println("Number = " + num);
        }       
    }
}

File: Julia_Q4_Sum.js
const vec = [];

while (true) {
    const input = prompt(`Input ${vec.length + 1} (or enter 'done' to finish):`);
  
    if (input === 'done') {
        break;
    }
  
    const number = parseInt(input);
    vec.push(number);
}

let sum = 0;
console.log('Array Output:');
vec.forEach((num, index) => {
    console.log(`Number ${index + 1} = ${num}`);
    sum += num;
});
console.log(`Sum = ${sum}`);
